modules/qna/src/views/full/FormModal.tsx

```diff
@@ -212,7 +215,21 @@ export default class FormModal extends Component<Props> {
     return this.state.item.questions[this.props.contentLang] || []
   }
 
-  createAnswer = answer => {
+  createAnswer = async answer => {
+    const textAnswers = await Promise.map(this.itemAnswers, (answer: string) =>
```

Comment:
Please change the variable `answer` to something like `a`, because it is the same as the outer variable name